music21/musicxml/m21ToXml.py

```diff
@@ -5654,6 +5660,32 @@ def wrapObjectInAttributes(self, objectToWrap, methodToMx):
 
         return mxAttributes
 
+    def _matches_previous_sibling_in_group(self,
```

Comment:
this is okay as underscore case as private, but the name should match the attribute.